client/urls.py

```diff
@@ -0,0 +1,10 @@
+"""urls for client application."""
+
+from django.urls import path
+
+from .views import client_view, client_detail_view
+
+urlpatterns = [
+    path('clients/', client_view, name='clients'),
```

Comment:
should be client, not clients. As we are handling one in a single call no multiple?